statsd/src/main/java/org/kaazing/monitoring/client/StatsdPublisher.java

```diff
@@ -115,21 +115,21 @@ private synchronized boolean flush() {
 
             // send and reset the buffer
             sendBuffer.flip();
-            final int nbSentBytes = _channel.send(sendBuffer, _address);
+            final int nbSentBytes = channel.send(sendBuffer, address);
             sendBuffer.limit(sendBuffer.capacity());
             sendBuffer.rewind();
 
             if (sizeOfBuffer == nbSentBytes) {
                 return true;
             } else {
-                log.error(String.format("Could not send entirely stat %s to host %s:%d. Only sent %d bytes out of %d bytes",
-                        sendBuffer.toString(), _address.getHostName(), _address.getPort(), nbSentBytes, sizeOfBuffer));
+                LOGGER.error(String.format("Could not send entirely stat %s to host %s:%d. Only sent %d bytes out of %d bytes",
+                        sendBuffer.toString(), address.getHostName(), address.getPort(), nbSentBytes, sizeOfBuffer));
```

Comment:
Does sending the incorrect number of bytes represent a complete loss of service, or could tis possibly be logged as a warning?